includes/api/class-edd-api-v2.php

```diff
@@ -145,7 +145,7 @@ public function get_products( $args = array() ) {
 			}
 		}
 
-		return apply_filters( 'edd_api_products_product_v2', $product );
+		return apply_filters( 'edd_api_products_product', $product );
```

Comment:
This should be `edd_api_products`. It's for a list of products, not a single product.